.github/ci-windows-latest.yaml

```diff
@@ -0,0 +1,92 @@
+
+# Just a work in progress version, not fully implemented. For conceptual purpose only.
+# Need to add more steps and tested to make it functional fully. But this can be achieved.  
+
+name: ci-windows-latest
+
+on:
+  push:
+    branches:
+      - 'main'
+      - 'master'
+
+jobs:
+  build-docker-latest:
+    runs-on: windows-2019
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v3
+      -
+        name: Login to DockerHub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
```

Comment:
is `secrets` the name that should be used here?

![image](https://user-images.githubusercontent.com/1633595/182855392-8dbf042d-fcfa-4882-97b5-001909876a84.png)